<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoviePageClass</class>
 <widget class="QDialog" name="MoviePageClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1001</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MoviePage</string>
  </property>
  <widget class="QPushButton" name="AddWishButton">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>260</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(17, 0, 255)</string>
   </property>
   <property name="text">
    <string>Add to Wishlist</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AddWatchedButton">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>210</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(17, 0, 255)</string>
   </property>
   <property name="text">
    <string>Watched</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="RateLineEdit">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>320</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="RateButton">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>320</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(17, 0, 255)</string>
   </property>
   <property name="text">
    <string>Rate the show</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_SimilarShows">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>440</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Here are some similar shows</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>470</y>
     <width>901</width>
     <height>221</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_Title">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>641</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_Rating">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>190</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Duration">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>190</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Score">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Description">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>641</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_Directors">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>641</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_Categories">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>641</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Countries">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>641</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Cast">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>641</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_ShowType">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="HomepageButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Go to Homepage</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_LastWatched">
   <property name="geometry">
    <rect>
     <x>838</x>
     <y>215</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
